code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -19,6 +19,10 @@
 	M.nutrition += nutriment_factor
 	holder.remove_reagent(src.id, metabolization_rate)
 
+/datum/reagent/consumable/reaction_mob(mob/living/carbon/M, method)
+	if(method == INGEST && reagent_state == LIQUID)
+		M.thirst += nutriment_factor * DRINK_THIRST_FACTOR
+
```

Comment:
So, the only way to get less thirsty is... to eat food...?

Reply:
reagent_state == LIQUID

Reply:
nutriment_factor

Reply:
drinks have nutriment factors

Reply:
yeah but not water

(probably)